### ISSUE TITLE
Chunk log information

### ISSUE THREAD

**Issue by vincentmanz** [LooseLab/ru#45]
Hey, 
Could we have clear information on the meaning of the chunk log file? 
Thanks

**Comment by mattloose** [comment]
Hi,

The chunk log has a header file. The column headings are:

client_iteration: The number of times we have grabbed data from the client.
read_in_loop: The read being handled in the chunk received from the client.
read_id: The read id of the read being processed.
channel: The channel on the flowcell this read is from.	
read_number: The read number of that read through that channel.
seq_len: The length of this specific piece of sequence.
counter: The number of times we have analysed this specific read.	
mode: How the read was classified.	
decision: The decision taken by interpreting the toml file for this read and mapping.
condition: The toml file rules that were applied.	
min_threshold: Has the minimum chunks rule been broken.
count_threshold: Has the maximum chunks rule been broken.
start_analysis: internal time stamp for starting analysis	
end_analysis: internal time stamp for ending analysis	
timestamp: Timestamp for handling this event.
